packages/insomnia-inso/src/cli.ts

```diff
@@ -64,6 +64,7 @@ const makeTestCommand = (commandCreator: CreateCommand) => {
     )
     .option('-b, --bail', 'abort ("bail") after first test failure')
     .option('--keepFile', 'do not delete the generated test file')
+    .option('--disableCertValidation', 'disable certificate validation')
```

Comment:
would it be more friendly to users viewing the help if we said:

```suggestion
    .option('--disableCertValidation', 'disable SSL certificate validation')
```

Reply:
Will defer to @wdawson on that!

Reply:
I created a ticket to make sure we address it: INS-830